chain/src/txhashset.rs

```diff
@@ -476,8 +476,7 @@ impl<'a> Extension<'a> {
 		let kernel_pos = self.kernel_pmmr.unpruned_size();
 
 		// Build bitmap of output pos spent (as inputs) by this tx for rewind.
-		let rewind_rm_pos = tx
-			.inputs()
+		let rewind_rm_pos = tx.inputs()
```

Comment:
I see PRs flip-flopping on this code style _all the time_ (first chained fn call on same line or next line).
Is this a difference between local rustfmt configs?

I'm on - 
```
rustfmt --version
rustfmt 0.8.2-stable (08da30d7 2018-06-06)
```

And I _think_ my rustfmt puts these on the next line which contradicts this PR.

Reply:
Agree, I think this is something related to https://github.com/rust-lang-nursery/rls-vscode/issues/210. Investigating.

Reply:
:+1: